bindings/pyroot/src/Pythonize.cxx

```diff
@@ -2691,14 +2691,14 @@ Bool_t PyROOT::Pythonize( PyObject* pyclass, const std::string& name )
 
    }
 
-   else if ( name.substr(0,8) == "TVector3" ) {
+   else if ( name.substr(0,8) == "RVector3" ) {
```

Comment:
Seems like a wrong replacement / change?

Reply:
Absolutely, this is a mistake.